xdsl/dialects/riscv.py

```diff
@@ -2064,29 +2065,22 @@ class DirectiveOp(IRDLOperation, RISCVOp):
     name = "riscv.directive"
     directive: StringAttr = attr_def(StringAttr)
     value: StringAttr | None = opt_attr_def(StringAttr)
```

Comment:
It feels like maybe we could make this non-optional now? I'm not 100% sure but it feels like if a directive doesn't demark a section in the assembly code then it should probably have a value. If it doesn't and the empty string is inappropriate the user can always create an assembly section with an empty region. But maybe I'm missing something.